packages/graphql/src/translate/cypher-builder/clauses/Return.ts

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright (c) "Neo4j"
+ * Neo4j Sweden AB [http://neo4j.com]
+ *
+ * This file is part of Neo4j.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Clause } from "./Clause";
+import type { NodeRef } from "../variables/NodeRef";
+import type { CypherEnvironment } from "../Environment";
+import { isString } from "../../../utils/utils";
+import type { Expr } from "../types";
+
+// TODO: make projection more elegant
```

Comment:
I like TODOs, but this comment isn't very helpful. Instead, I think that if there is a feeling that something can be improved in this code then writing more specifically about what those improvements would be will be more helpful.